include stdint.h for uint_XX

Fixes
task_runner.h:48:55: error: unknown type name 'uint32_t'
  virtual void PostDelayedTask(std::function<void()>, uint32_t delay_ms) = 0;
                                                      ^
1 error generated.

Upstream-Status: Pending
Signed-off-by: Khem Raj <raj.khem@gmail.com>
--- a/third_party/perfetto/include/perfetto/base/task_runner.h
+++ b/third_party/perfetto/include/perfetto/base/task_runner.h
@@ -17,6 +17,7 @@
 #ifndef INCLUDE_PERFETTO_BASE_TASK_RUNNER_H_
 #define INCLUDE_PERFETTO_BASE_TASK_RUNNER_H_
 
+#include <cstdint>
 #include <functional>
 
 #include "perfetto/base/export.h"
--- a/third_party/webrtc/call/rtx_receive_stream.h
+++ b/third_party/webrtc/call/rtx_receive_stream.h
@@ -11,6 +11,7 @@
 #ifndef CALL_RTX_RECEIVE_STREAM_H_
 #define CALL_RTX_RECEIVE_STREAM_H_
 
+#include <cstdint>
 #include <map>
 
 #include "call/rtp_packet_sink_interface.h"
